validation_run.py

```diff
@@ -0,0 +1,182 @@
+# This code is used to generate simulations using parameters from the single outbreaks. Each set of parameters will generate 5000 valid results (We only consider the simulations with number of introductions on day 1 > 0)
+# Please note that the code below requires significant amount of computational resources since in order to generate valid results, the number of total simulations can reach to 10E13.
+
+## import libraries
+import numpy as np
+import pandas as pd
+
+from collections import namedtuple
+from itertools import product
+from scipy.stats import binom
+from scipy.stats.mstats import gmean
+from scipy.interpolate import Rbf
+
+from sklearn.preprocessing import power_transform
+import math
+import pickle
+import itertools
+
+import importlib
+import statsmodels.api as st
+import sys
+
+## import code
+import covid_workplace_model as new_model
+
+print("version: ", new_model.version())
+
+######## Parameter input ##########
+######## Please comment out unused parameters ############
+
+# ## Tianjin office
+# print("*"*5, 'Tianjin office', '*'*5)
+# EMPLOYEE=906
+# WFH=0
+# CASE_RATE=1/100000
+# DAILY_AVERAGE_CONTACTS=[4,10]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.05,.25]
+# NUMBER_DAYS=10
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=7
+
+## Korean call center
+print("*" * 5, "Korean Call Center", "*" * 5)
```

Comment:
This seems unnecessary since the file does not print the outputs.

Reply:
I removed this